src/js/components/PageSection/PageSection.js

```diff
@@ -0,0 +1,27 @@
+import React, { useContext } from 'react';
+import { Box } from '../Box';
+import { PageContext } from '../Page';
+import { PageSectionPropTypes } from './propTypes';
+
+const PageSection = ({ background = 'none', children, full, ...rest }) => {
+  const { ...pageContext } = useContext(PageContext);
+  if (full) {
+    return (
+      <Box background={background}>
+        <Box {...pageContext} {...rest}>
+          {children}
+        </Box>
+      </Box>
+    );
+  }
+  return (
+    <Box background={background} {...pageContext} {...rest}>
+      {children}
+    </Box>
+  );
+};
```

Comment:
Could also do something like this. Set up the base, then add the wrapper if needed. That way if we need to modify the base, we do it in a single location.

```suggestion
  let contents = <Box background={background} {...pageContext} {...rest}>
      {children}
    </Box>;
  if (full) {
    contents = <Box background={background}>
        {contents}
      </Box>;
  }
  return contents;
};
```